README.md

```diff
@@ -60,10 +63,53 @@ The bundle will deploy:
 - 1 `coordinator` unit (`tempo-coordinator-k8s` charm)
 - 1 `s3-integrator` unit (`s3-integrator` charm)
 
-[note]
+[!NOTE]  
 The default number of worker units is an odd number to prevent split-brain situations, and greater than 1 for the deployment to be HA.
-[/note]
 
+### Minimal microservices deployment
+The bundle is able to deploy Tempo in its [microservices mode](https://grafana.com/docs/tempo/latest/setup/deployment/#microservices-mode), which runs each one of the required roles in distinct processes. Each required role is assigned to one worker unit to achieve the minimal deployment for the cluster to be considered consistent. Minimal microservices mode is the way to deploy Grafana Tempo in a distributed architecture without high availability.
+
+Add `--mode` to deploy `tempo-bundle` in `minimal-microservices` mode.
+
+```shell
+# render bundle with "edge" charms
+tox -e render-edge -- --mode=minimal-microservices
+juju deploy ./bundle.yaml --trust
+```
+
+The bundle will deploy:
+
+- 1 `tempo-worker-querier` unit (`tempo-worker-k8s` charm)
+- 1 `tempo-worker-query-frontend` unit (`tempo-worker-k8s` charm)
+- 1 `tempo-worker-ingester` unit (`tempo-worker-k8s` charm)
+- 1 `tempo-worker-distributor` unit (`tempo-worker-k8s` charm)
+- 1 `tempo-worker-compactor` unit (`tempo-worker-k8s` charm)
+- 1 `tempo-worker-metrics-generator` unit (`tempo-worker-k8s` charm)
+- 1 `coordinator` unit (`tempo-coordinator-k8s` charm)
+- 1 `s3-integrator` unit (`s3-integrator` charm)
+
+### Recommended microservices deployment
+The bundle is able to deploy Tempo in its [microservices mode](https://grafana.com/docs/tempo/latest/setup/deployment/#microservices-mode), which runs each one of the required roles in distinct processes. Each required role is assigned to an arbitrary number of worker units to achieve the recommended deployment for the cluster to be considered robust. Recommended microservices mode is the way to deploy Grafana Tempo in a distributed architecture while achieving high availability in a production environment.
```

Comment:
```suggestion
Finally, the bundle can be rendered to deploy the 'recommended' amount of nodes for its microservices mode. In this scenario, each worker node is scaled up to the recommended number of units. This is the recommended way to deploy a highly available Grafana Tempo for production environments.
```

Should we add a note/link explaining who is doing the recommendation? Is this according to some official grafana guide, helm chart, or is this our expert opinion?

Reply:
Its still a TODO item I guess to find out what does the recommended deployment look like

Reply:
I see grafana have their helm chart for tempo with these values https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/values.yaml
Guessing we can have this as our initial recommendation at least until we have a counter argument.